### ISSUE TITLE
Customize route color behind the puck issue

### ISSUE THREAD

**Issue by AbdullahSoubeih** [mapbox/mapbox-navigation-android#3387]
Can't resolve routeLineTraveledColor and routeLineShieldTraveledColor on navigation-ui:0.42.6

**Comment by Guardiola31337** [comment]
@abhishek1508 Could you confirm is this is still reproducible in `1.0`?

**Comment by abhishek1508** [comment]
These attributes are not available in `navigation-ui:0.42.6`. @AbdullahSoubeih please bump your version to `1.0.0-rc.6` to access these properties. These properties are referred to as 
```
<item name="routeLineTraveledColor">@color/mapbox_navigation_route_line_traveled_color</item>
<item name="routeLineCasingTraveledColor">@color/mapbox_navigation_route_casing_line_traveled_color</item>
```

cc @Guardiola31337

**Comment by abhishek1508** [comment]
Closing the ticket as this is not an issue. 
@AbdullahSoubeih please feel free to open a new one if you still have issues with these properties in `1.0.0-rc.6` which is our current public version. We are still working on releasing the documentation for the same.
